src/d3a/models/strategy/__init__.py

```diff
@@ -204,14 +204,14 @@ def is_eligible_for_balancing_market(self):
 
     def accept_offer(self, market: Market, offer, *, buyer=None, energy=None,
                      already_tracked=False, trade_rate: float = None,
-                     original_trade_rate: float = None):
+                     trade_bid_info: float = None):
```

Comment:
Minor: How about renaming this parameter to `trade_offer_bid_info` ?

Reply:
Because this should contain only the bid info. The offer info have to be retrieved from this market.